package.json

```diff
@@ -26,74 +26,76 @@
   },
   "dependencies": {
     "classnames": "^2.2.5",
+    "libphonenumber-js-utils": "^8.9.9",
     "prop-types": "^15.6.1",
     "react-style-proptype": "^3.0.0",
     "underscore.deferred": "^0.4.0"
   },
   "devDependencies": {
-    "babel-core": "^6.21.0",
-    "babel-eslint": "^7.1.1",
-    "babel-jest": "^18.0.0",
-    "babel-loader": "6.2.10",
-    "babel-plugin-transform-class-properties": "^6.19.0",
-    "babel-plugin-transform-object-rest-spread": "^6.20.2",
-    "babel-plugin-transform-react-constant-elements": "^6.9.1",
-    "babel-plugin-transform-react-remove-prop-types": "^0.4.13",
-    "babel-plugin-transform-regenerator": "^6.21.0",
-    "babel-plugin-transform-runtime": "^6.15.0",
-    "babel-plugin-dynamic-import-node": "^2.0.0",
-    "babel-plugin-syntax-dynamic-import": "^6.18.0",
-    "babel-preset-env": "^1.1.8",
-    "babel-preset-react": "^6.16.0",
-    "babel-preset-react-app": "^2.0.1",
-    "babel-runtime": "^6.20.0",
-    "case-sensitive-paths-webpack-plugin": "^1.1.4",
+    "@babel/core": "^7.2.0",
+    "@babel/plugin-proposal-class-properties": "^7.2.1",
+    "@babel/plugin-syntax-dynamic-import": "^7.2.0",
+    "@babel/polyfill": "^7.0.0",
+    "@babel/preset-env": "^7.2.0",
+    "@babel/preset-react": "^7.0.0",
+    "babel-core": "^7.0.0-bridge.0",
+    "babel-eslint": "^10.0.1",
+    "babel-jest": "^23.6.0",
+    "babel-loader": "^8.0.4",
+    "babel-plugin-dynamic-import-node": "^2.2.0",
+    "case-sensitive-paths-webpack-plugin": "^2.1.2",
     "chalk": "1.1.3",
-    "copy-webpack-plugin": "^4.0.1",
+    "copy-webpack-plugin": "^4.6.0",
     "coveralls": "^2.11.9",
-    "css-loader": "^0.26.1",
+    "css-loader": "^1.0.1",
     "css-modules-require-hook": "^4.0.1",
     "detect-port": "^1.0.7",
     "dotenv": "^4.0.0",
     "enzyme": "^3.3.0",
-    "enzyme-adapter-react-16": "^1.1.1",
-    "eslint": "^3.13.0",
-    "eslint-config-airbnb": "~14.0.0",
-    "eslint-config-airbnb-base": "~11.0.1",
-    "eslint-loader": "1.6.1",
-    "eslint-plugin-flowtype": "^2.29.2",
-    "eslint-plugin-import": "^2.0.1",
-    "eslint-plugin-jsx-a11y": "^3.0.2",
-    "eslint-plugin-react": "^6.9.0",
-    "eslint-plugin-security": "^1.2.0",
-    "extract-text-webpack-plugin": "^2.1.0",
-    "file-loader": "^0.9.0",
+    "enzyme-adapter-react-16": "^1.5.0",
+    "eslint": "^5.3.0",
+    "eslint-config-airbnb": "~17.1.0",
+    "eslint-config-airbnb-base": "~13.1.0",
```

Comment:
Can we use `^` instead of `~`

Reply:
To avoid some new lint rules in the future breaks the build, I think we should just lock with `~`.